### PR TITLE
BricLayer feature work

### PR DESCRIPTION
**Pull request by mlippert**
This is the review pull request for the BricLayer feature now that I've pushed my initial commit.
It should not be merged until I request a code review and the code reviewer agrees it's ready.

Currently all I've got is a skeleton BricLayer class that creates a BricWorks instance and initializes it.

There are warnings to be removed even from this small piece of code.

### CODE REVIEW

**Comment by mlippert** [comment]
I think this finishes off the reduced 445 (BricLayer) story.
I've asked @seannives and @ysahnpark for a code-review before I merge back to master

File: js/briclayer.js [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+ *
+ ****************************************************************************/
+pearson.brix.BricLayer.prototype.build = function (activityConfig)
+{
+    /**
+     * @dict
+     */
+    var building = {'info': {}, 'brix': {}, 'mortar': {}};
+
+    building['info']['seqNodeId'] = activityConfig['sequenceNodeKey'];
```
Comment (seannives) [inline js/briclayer.js]: can this be changed to building['info']['sequenceNodeKey']?

File: js/briclayer.js [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+     *
+     * An action which calls a method of a particular instance object with a
+     * given set of arguments.
+     *
+     * @this {pearson.brix.BricLayer}
+     * @param {Object}  building    -the under construction (by the build method) building
+     * @param {Object}  action       -the action object
+     *
+     ****************************************************************************/
+    'method-call': function (building, action)
```
Comment (seannives) [inline js/briclayer.js:397]: this is some ninja shit right here.  :-)

File: js/widget-numeric.js [hunk 3]
```diff
@@ -83,18 +91,18 @@ pearson.brix.Readout = function (config)
...
 		.attr("class", "dataLabel")
-		.attr("id", this.id)
+		.attr("id", this.readoutId_)
 		.text(this.startVal);
 
 	this.rootEl.append("span").html("&nbsp;" + (this.unit ? this.unit : ""));
 	
 	// TEST: the written text span is the start value
-	window.console.log("text is the startVal:", d3.select("#" + this.id).text() == this.startVal, d3.select("#" + this.id).text());
+	window.console.log("text is the startVal:", d3.select("#" + this.readoutId_).text() == this.startVal, d3.select("#" + this.readoutId_).text());
```
Comment (seannives) [inline js/widget-numeric.js:105]: Do we still need these console logs?

**Comment by seannives** [comment]
:+1:

File: js/briclayer.js (hunk 1 above)
Comment (mlippert) [inline js/briclayer.js]: yes, I'll do that before I merge.

File: js/widget-numeric.js (hunk 3 above)
Comment (mlippert) [inline js/widget-numeric.js:105]: Well, I wasn't changing functionality here, so you'll have to wait until that happens. All I was doing was trying to get some properties named in a way that will be more friendly to javascript inheritance.

**Comment by ysahnpark** [comment]
:+1:

**Pull request merged**